CHANGELOG.md

```diff
@@ -21,8 +21,9 @@ Adding a requirement of a major version of a dependency is breaking a contract.
 Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
-[Unreleased]: https://github.com/atlassian/docker-infrastructure/compare/release-0.3.6...master
+[Unreleased]: https://github.com/atlassian/docker-infrastructure/compare/release-0.3.7...master
 
+## [0.3.7] - 2023-09-11
```

Comment:
Missing link to the `0.3.7`. Also missing for 2 previous releases.
```suggestion
## [0.3.7] - 2023-09-11
[0.3.7]: https://github.com/atlassian/docker-infrastructure/compare/release-0.3.6...release-0.3.7
```

Reply:
Weird... the last tag we have is `0.3.5`, so we don't have `0.3.6` and `0.3.7`

Reply:
Something went wrong in the last release: https://github.com/atlassian/docker-infrastructure/actions/runs/6148762797/job/16683636787#step:4:64

Reply:
Fixed